server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -24,18 +21,15 @@
 import org.opensearch.index.shard.IndexShard;
 import org.opensearch.indices.recovery.FileChunkRequest;
 import org.opensearch.indices.replication.common.ReplicationType;
-import org.opensearch.rest.RestStatus;
 import org.opensearch.test.BackgroundIndexer;
 import org.opensearch.test.InternalTestCluster;
 import org.opensearch.test.OpenSearchIntegTestCase;
 import org.opensearch.test.transport.MockTransportService;
 import org.opensearch.transport.TransportService;
 
-import java.util.ArrayList;
```

Comment:
```
/home/runner/work/OpenSearch/OpenSearch/server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java:293: error: cannot find symbol

        List<String> nodes = new ArrayList<>();
```

Reply:
[This commit](https://github.com/opensearch-project/OpenSearch/commit/ce67ca603d41ef59ed22d1ce357a172bcc080c8e) added an ArrayList so it depends on this import.